### PR TITLE
[Core] Delete Archive

### PR DESCRIPTION
**Pull request by Elyniss**
Merge after: https://github.com/WalletConnect/WalletConnectKotlinV2/pull/1143

### CODE REVIEW

File: protocol/push/src/main/kotlin/com/walletconnect/push/di/SyncInPushModule.kt [hunk 1]
```diff
@@ -43,6 +42,6 @@ internal fun syncInPushModule() = module {
         )
     }
 
-    single { GetMessagesFromHistoryUseCase(get(),get(), get()) }
+    single { GetMessagesFromHistoryUseCase(get(),get()) }
```
Comment (TalhaAli00) [inline protocol/push/src/main/kotlin/com/walletconnect/push/di/SyncInPushModule.kt:45]: let's add named arguments to all usecases

**Review by TalhaAli00 — APPROVED** [review]

File: protocol/push/src/main/kotlin/com/walletconnect/push/di/SyncInPushModule.kt (hunk 1 above)
Comment (Elyniss) [reply protocol/push/src/main/kotlin/com/walletconnect/push/di/SyncInPushModule.kt:45]: This module will be removed in following PR

File: protocol/push/src/main/kotlin/com/walletconnect/push/engine/sync/use_case/GetMessagesFromHistoryUseCase.kt [hunk 2]
```diff
@@ -27,14 +24,7 @@ internal class GetMessagesFromHistoryUseCase(
...
             syncClient.getStoreTopic(Sync.Params.GetStoreTopics(accountId, store.value))?.let { topic ->
-                archiveInterface.getAllMessages(
-                    MessagesParams(topic.value, null, ArchiveInterface.DEFAULT_BATCH_SIZE, null),
-                    onError = { error -> onError(error.throwable) },
-                    onSuccess = {
-                        logger.log("Fetched ${it.size} for ${store.value}")
-                        countDownLatch.countDown()
-                    }
-                )
+                // Do nothing
```
Comment (jakubuid) [inline protocol/push/src/main/kotlin/com/walletconnect/push/engine/sync/use_case/GetMessagesFromHistoryUseCase.kt:27]: Remove

**Review by jakubuid — APPROVED** [review]

File: protocol/push/src/main/kotlin/com/walletconnect/push/engine/sync/use_case/GetMessagesFromHistoryUseCase.kt (hunk 2 above)
Comment (Elyniss) [reply protocol/push/src/main/kotlin/com/walletconnect/push/engine/sync/use_case/GetMessagesFromHistoryUseCase.kt:27]: Will be removed in next PR

**Comment by Elyniss** [comment]
Not sure why all of a sudden Github "dissmised stale reviews" merging anyway as I only changed base, team was aware of it

**Pull request merged**